lacommunaute/users/models.py

```diff
@@ -1,9 +1,24 @@
-from django.contrib.auth.models import AbstractUser
+from uuid import uuid4
+
+from django.contrib.auth.models import AbstractUser, UserManager as BaseUserManager
 from django.db import models
 
 from lacommunaute.users.enums import IdentityProvider
 
 
+class UserManager(BaseUserManager):
+    def generate_unique_username(self):
+        username = uuid4().hex
+        while User.objects.filter(username=username).exists():
+            username = uuid4().hex
+        return username
```

Comment:
```suggestion
        return str(uuid4())
```

Je pense que c'est overkill, il y aura d'autres soucis avant une collision.

Si tu cast en `str` tu as les `-` et les groupes, pref personnelle, mais tu peux tout à fait garder `uuid4().hex`